### ISSUE TITLE
Please release kernel source for NX659J Q as soon as possible!  Devs are waiting on it.

### ISSUE THREAD

**Issue by mrslezak** [ztemt/NX627J_P_kernel#1]
Please release kernel source for NX659J Q (Redmagic 5G) as soon as possible!  Devs are waiting on it.  The last open source release we see here is for the Redmagic 3 Pie, which is severely outdated.  People are already unlocking bootloaders: https://forum.xda-developers.com/nubia-red-magic-5g/how-to/how-to-unlock-bootloader-redmagic-5g-t4081743 but the unlock process breaks the FP sensor.  So we need that fixed as well.  Thank you for doing this quickly!  You will not regret it, good things will be coming to Nubia!!!!

**Comment by yamvmax** [comment]
Please release kernel source for Redmagic 5g as soon as possible. Fingerprint sensor is broken when bootloader is unlocked!

**Comment by yazidhs** [comment]
Hello, please we do need you to release the kernel source for A10 on the Redmagic 5G (NX659J).

Please do not go down the same path with redmagic 3 and 3s.

This is a great hardware, please give it a good software and developers support and you'll  have a customer base you didn't project. 

Give us software support!!!

**Comment by samaparr** [comment]
Please release the kernal source code! 
This is very important for developers and will help build a create community around the Red Magic 5G.

Also please fix the fingerprint unlock after unlocking the bootloader! 

Thanks.

**Comment by mendezgiovanny** [comment]
Please release kernel source for NX659J Q as soon as possible! Devs are waiting on it.

**Comment by fandango96** [comment]
Just wanted to echo the previous comments. Please prioritize the kernel release of the international version over the Chinese version if you do have to choose, but having both would be awesome. I would love to use LineageOS 17.x on this device and wouldn't mind working on porting it myself.

**Comment by ace52182** [comment]
Please release the source code for Redmagic 5G (NX659J). Developers are waiting and this could only be a plus for us Nubia consumers and Nubia as well as it will only strengthen the Redmagic brand by expanding the community. It would also be appreciated if you could fix the fingerprint issue when the bootloader is unlocked in your next update or shortly after.Thank You Kindly.. A Red Magic 5g owner...

**Comment by gokhujee** [comment]
Please release the source code for red magic 5g, this will be excellent for your sales as more people buy and more people will enjoy there device, also please fix fingerprint for when unlocking your device, you have a choice we shouldn't be restricted, please fix this on new update , thanks so much .

**Comment by apersomany** [comment]
Yeah, the devs are waiting for the source code. Please release it.

**Comment by qromzi** [comment]
By releasing the source the development will be much faster amd the phone will be more popular and user friendly. Please release the kernel source.

**Comment by Sightrasher** [comment]
Please release the source Code for RM5G, there is a whole community ready to customize and improve the device (and your sales lol). Thank you in advance :)

**Comment by enyceckk** [comment]
Please release kernel source for NX659J.. this device has a lot of potential.. just need a few tweaks to make this phone even better.

**Comment by SickSecCo** [comment]
Please solve NX659J fingerprint that doesn't work after unlocking the bootloader, only with your help can a community be created that contributes to the improvement of the smartphone.
Moreover releasing the source code can only be good for the brand and the development of this.

**Comment by bAN0NYM0US** [comment]
Please release the kernel source for NX659J, I was going to return my phone until I found people also trying to get this figured out, I'm holding off now with hope that this will become available.

**Comment by Velemir1982** [comment]
Please release kernel source for NX659J Q as soon as possible! The only alternative is that you will fix the software issues by yourself ...

**Comment by gesangtome** [comment]
Why not update it?
Release the complete kernel source Pls

**Comment by boom8i** [comment]
Please release.  REDMAGIC 5G TWRP PROJECT  PLZ
